docs/compression_algorithms/Pruning.md

```diff
@@ -27,6 +27,46 @@ Where ![L_j](https://latex.codecogs.com/png.latex?L_j) is j-th convolutional lay
 
 Then during pruning filters with smaller ![G(F_i)](https://latex.codecogs.com/png.latex?G%28F_i%29) importance function will be pruned first.
 
+#### Algorithm statistics
+A model compression can be measured by two main metrics: filter pruning level and FLOPs pruning level. While 
+filter pruning level shows the ratio of removed filters to the total number of filters in the model, FLOPs pruning level 
+indicates how the removed filters affect the number of floating point operations required to run a model. 
+
+During the algorithm execution several compression statistics are available:
+
+`Filter pruning level` - percentage of filters removed from the model. Calculated as: 
+
+![Filter pruning level](https://latex.codecogs.com/png.image?%5Cdpi%7B110%7D%20%5Cinline%20%5Ctextit%7Bfilter%20pruning%20rate%20%7D=%201%20-%20(%5Ctextit%7Bcurrent%20number%20of%20filters%20in%20the%20model%20%7D/%5Ctextit%7B%20total%20number%20of%20filters%20in%20the%20model%7D))
+
+> **NOTE**: All other pruning levels are calculated in a similar way.
```

Comment:
Please consider to make description of statistic pruning level general as it done in pruning stats